index.html

```diff
@@ -50,6 +50,135 @@ <h1>I’m Eugene. <br>
+                <div class="card">
+                    <img src="images/Snapshoot Portfolio (7).svg" alt="">
+                </div>
+                <div class="left-block">
+                    <h2>Multi-Post Stories</h2>
+                    <ul class="position">
+                        <li>CANOPY<i class="fa fa-circle"></i></li>
+                        <li>Back End Dev<i class="fa fa-circle"></i></li>
+                        <li>2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="programming-lang">
+                        <li><a href="#">HTML</a></li>
+                        <li><a href="#">CSS</a></li>
+                        <li><a href="#">Javascript</a></li>
+                    </ul>
+                    <button class="btn-action">See project</button>
+                </div>
+            </div>
+        </div>
+        <div class="about-myself">
+          <div class="about-container">
+            <h1>About Myself</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <a href="#" class="connect">
+                LET'S CONNECT
+            </a> 
+            <ul class="social-media">
+                <li><a href=""><img src="images/portfolio1.svg" alt=""></a></li>
+                <li><a href=""><img src="images/Linkedin icon.svg" alt=""></a></li>
+                <li><a href=""><img src="images/portfolio4.svg" alt=""></a></li>
+                <li><a href=""><img src="images/portfolio3.svg" alt=""></a></li>
+                <li><a href=""><img src="images/portfolio5.svg" alt=""></a></li>                       
+            </ul>
+            <button class="btn-action">Get my resume</button>
+          </div>
+          <div class="skill-container">
+            <div class="languages">
+                <h2>Languages</h2>
+                <i class="fa fa-chevron-down"></i>
+            </div>
+            <ul class="language-list">
+                <li>
+                    <img src="images/Ellipse 1.png" alt="Javascript">
+                    <h2>Javascript</h>
+                </li>
+                <li>
+                    <img src="images/Ellipse 1 (2).png" alt="HTML">
+                    <h2>HTML</h2>
+                </li>
+                <li>
+                    <img src="images/Ellipse 1 (3).png" alt="CSS">
+                    <h2>CSS</h2></li>
+            </ul>
+            <ul class="frames">
+                <li>
+                    <span><h2>Frameworks</h2><i class="fa fa-chevron-right"></i></span>
+                </li>
+                <hr/>
+                <li>
+                    <span><h2>Skills</h2><i class="fa fa-chevron-right"></i></span>
+                </li>
```

Comment:
- [ ] (optional) Kindly consider following the given design by using the correct colors provided for the `Languages` text which is `#253858`. 
- [ ] (optional) Also consider using the given font size

Given design
![image](https://github.com/Eugeneiyukhun/Setup_and_Mobile_First/assets/81932478/b7d278b4-7bc6-478f-9709-f97a938f2b1c)

Your design
![image](https://github.com/Eugeneiyukhun/Setup_and_Mobile_First/assets/81932478/88606c2f-5b86-4eb6-bc2d-8568e90b3c0f)